README.md

```diff
@@ -199,12 +199,12 @@ You have {numPhotos, plural,
 ```
 
 ```hbs
-{{format-message (intl-get 'product.info')
+{{t 'product.info'
```

Comment:
Are in intending to deprecate `format-message` and move to just `t` going forward? If not, I'd keep at least this example with the intl-get somewhere in the docs so people know how to use it should they choose.